src/test/run-make/issue-26995/Makefile

```diff
@@ -0,0 +1,6 @@
+-include ../tools.mk
+
+all:
+	printf "#[path=\"%s/%s\"] pub mod baz;\n" `pwd` "bar/baz.rs" > foo.rs
```

Comment:
Perhaps this could be a unix-only rustdoc test which has a module with the path as `/dev/null`? That would avoid run-make and you could still check for the module page, right?